README.md

```diff
@@ -12,73 +14,70 @@
+1. [Light Nodes](#celestia-light-nodes) (low CPU, 5GB+ disk): [get started](#run-a-light-node)
+2. [Bridge Nodes](#celestia-validator-bridge-nodes) (low CPU, 100GB+ disk): [get started](#run-a-bridge-node)
+3. [Bridge Validator Nodes](#celestia-validator-bridge-nodes) (same node as 2): [get started](#run-a-validator-bridge-node)
+4. _[Full DA Nodes](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-003-march2022-testnet.md#full-node) are under development_
 
 You can view chain activity at the current `devnet-2` explorer: https://celestia.observer/
 ## Overview
 
-Devnet demonstrates Celestia’s data availability capabilities on a libp2p network and live blockchain. 
+Devnet demonstrates Celestia’s data availability capabilities on a libp2p network and a syncing blockchain. 
 
-The current Devnet implementation runs two individual but connected networks:
+The Devnet implementation actually runs two individual but connected networks:
 
-1. A libp2p network with [**Light Nodes**](#celestia-light-nodes), that handles data availability interactions
-2. A p2p network with [**Validators Nodes**](#bridge-validator-nodes), that handles the underlying consensus and block production. 
+![Network Overview](diagrams/NetworkOverview.png)
 
-Special [**Bridge Nodes**](#bridge-nodes) process blocks from the underlying consensus network to the data availability network.
+1. A libp2p **DA network** with [Bridge Nodes](#bridge-nodes) which process blocks from the underlying p2p consensus network and [Light Nodes](#celestia-light-nodes), which do data availability sampling on Bridge Nodes' blocks.
+2. A p2p **Consensus network** with [**Validators Nodes**](#bridge-validator-nodes) (which are part of Bridge Nodes), that handles the underlying consensus and block production. 
 
-![Network Overview](diagrams/NetworkOverview.png)
+> Note that mainnet may look very different from this devnet implementation, as the architecture continues to be improved. You can read more about devnet decisions here ([see ADR](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-003-march2022-testnet.md)).
 
-> It’s important to note that mainnet may look very different from this devnet implementation, as the architecture continues to be improved. You can read more about devnet decisions here and here (link to ADR). 
+### Celestia (Validator) Bridge Nodes
+![Bridge Nodes](diagrams/BridgeNodes.png)
+Specifically, Bridge Nodes: 
 
-## Celestia Light Nodes
+1. Import and process “raw” headers & blocks from a trusted Core process in the Consensus network. Bridge Nodes can run this Core process internally or simply connect to a remote endpoint. Bridge Nodes also have the option of being an active validator in the Consensus network.
+2. Validate and erasure code the "raw" blocks
+3. Produce block shares with DA metadata to Light Nodes in the DA network.
```

Comment:
```suggestion
3. Supply block shares with DA metadata to Light Nodes in the DA network.
```

Reply:
What is DA metadata in this case?

Reply:
I'll change it to say DataAvailabilityHeaders, instead of "metadata", as per: 
```
type ExtendedHeader struct {
	RawHeader    `json:"header"`
	Commit       *core.Commit               `json:"commit"`
	ValidatorSet *core.ValidatorSet         `json:"validator_set"`
	DAH          *da.DataAvailabilityHeader `json:"dah"`
}
```